docs/pages/docs/avatar.mdx

```diff
@@ -23,10 +37,53 @@ An image element with a fallback for representing the user.
 ### `FAvatar(...)`
 
 ```dart
+// Default constructor - Valid image
 FAvatar(
-  image: const NetworkImage('https://picsum.photos/250?image=9'),
-  size: 60,
-  placeholderBuilder: (_) => const Text('MN'),
+  image: const NetworkImage('https://raw.githubusercontent.com/forus-labs/forui/main/samples/assets/avatar.png'),
+  placeholder: const Text('MN'),
+),
+
+// Default constructor - Invalid image with placeholder
+FAvatar(
+  image: const NetworkImage(''),
+  placeholder: const Text('MN'),
+),
```

Comment:
I don't think we need to include the invalid version here. Feel free to add the comments in the example above though.